README.adoc

```diff
@@ -854,14 +854,14 @@ Consider using named placeholders instead of positional placeholders when you ha
 ----
 # okish
 Client.where(
-  'created_at >= ? AND created_at <= ?',
-  params[:start_date], params[:end_date]
+  'orders_count >= ? AND country_code = ?',
+  params[:min_orders_count], params[:country_code]
 )
 
 # good
 Client.where(
-  'created_at >= :start_date AND created_at <= :end_date',
-  start_date: params[:start_date], end_date: params[:end_date]
+  'orders_count >= :min_orders_count AND country_code = :country_code',
+  min_orders_count: params[:min_orders_count], country_code: params[:country_code]
```

Comment:
If this change is not related to the addition of `Where with Ranges` new rule, it could be opened as a separate PR.

Reply:
On a second thought, I've made this change to make Named Placeholder example not to go against the new Where with Ranges.
Without this, it would have to be written as:
```ruby
Client.where(created_at: params[:start_date]..params[:end_date])
```
which defeats the purpose of being a good example for Named Placeholder.